packages/bar/src/Bar.tsx

```diff
@@ -362,6 +404,23 @@ const InnerBar = <RawDatum extends BarDatum>({
         )
     }
 
+    if (layers.includes('totals') && enableTotals) {
+        layerById.totals = (
+            <Fragment key="totals">
+                {totalsTransition((style, barTotal) =>
+                    createElement(BarTotal, {
```

Comment:
As the component is not customizable, we can use JSX directly rather than `createElement`, probably easier for typings.